src/assets/script/linkedin/linkedin.tsx

```diff
@@ -0,0 +1,18 @@
+import { type IIconsProps } from "types/common/PageProps";
+
+export const Linkedin = ({ colorArrow = "#191919", classNames }: IIconsProps) => (
```

Comment:
better create in src/helpers/colorPalette.ts colorPalette objects wittch have necessary colors from css root. can you correct it here and in other similar places?